src/audio/smart_amp/smart_amp.c

```diff
@@ -111,8 +111,11 @@ static ssize_t smart_amp_alloc_mod_memblk(struct smart_amp_data *sad,
 
 	/* query the required size from inner model. */
 	ret = mod->mod_ops->query_memblk_size(mod, blk);
```

Comment:
@johnylin76 FYI we are exploring 64bit builds now so sizes might differ between builds. It might be wiser to have the API use `ssize_t` rather than `int`. Minor nit for you to consider.